### PR TITLE
Allow the code to run from other package.

### PR DESCRIPTION
**Pull request by plural**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -6,7 +6,9 @@ function makeReader(filename: string) {
 
 	return function(): Record<string, any>[] {
 		if (json.length === 0) {
-			json = JSON.parse(fs.readFileSync(resolve('.', filename + '.json'), 'utf-8'));
+			let path = resolve(__dirname, '..', filename + '.json');
+			console.log('path is ' + path);
```
Comment (NoahTheDuke) [inline src/index.ts:10]: We should probably use a logging tool here, so consumers of the API don't have messages they don't want.

File: src/index.ts [hunk 2]
```diff
@@ -19,11 +21,15 @@ export const getMwlJson = makeReader('mwl');
...
+export function getDirName(): string {
+		return resolve(__dirname, '..', 'pack');
+}
+
 export function getCardsJson(): Record<string, any>[] {
 	const CARDS_JSON: Record<string, any>[] = [];
 
 	if (CARDS_JSON.length === 0) {
-		const directory = resolve('.', 'pack');
+		const directory = resolve(__dirname, '..', 'pack');
```
Comment (NoahTheDuke) [inline src/index.ts:32]: Use `getDirName` here?

File: src/index.ts (hunk 1 above)
Comment (NoahTheDuke) [reply src/index.ts:10]: the package `loglevel` is really good for this, tbh

Comment (plural) [reply src/index.ts:10]: yeah, this was just me being sloppy at the end of the day.  This and the other method were there just to confirm what i thought was happening in the other repo.

File: src/index.ts (hunk 2 above)
Comment (plural) [reply src/index.ts:32]: deleted the function.

**Review by NoahTheDuke — APPROVED** [review]

**Pull request merged**